src/containerapp/azext_containerapp/_clients.py

```diff
@@ -492,3 +492,106 @@ def list(cls, cmd, resource_group_name, env_name, formatter=lambda x: x):
                 env_list.append(formatted)
 
         return env_list
+
+
+class BuilderClient():
```

Comment:
If only `BuilderClient` need api-version 2023-08-01-preview in this feature, I recommend you only update the version for BuilderClient.
For example: 
`api_version = "2023-08-01-preview"`

Reply:
For example:
`BUILDER_CLIENT_API_VERSION = "2023-08-01-preview"`

Reply:
Thanks, updating this accordingly so that only BuildClient and BuilderClient get the new API version